tensorflow/core/kernels/BUILD

```diff
@@ -38,8 +37,11 @@ load(
     "if_mkl_ml",
     "mkl_deps",
 )
-load("@local_config_cuda//cuda:build_defs.bzl", "if_cuda", "if_cuda_is_configured")
-load("@local_config_rocm//rocm:build_defs.bzl", "if_rocm", "if_rocm_is_configured")
+load("@local_config_cuda//cuda:build_defs.bzl", "if_cuda")
+load(
```

Comment:
@jeffdaily could you run this file with `buildifier -v -mode=check`? Please refer to `tensorflow/tools/ci_build/ci_sanity.sh`.

Reply:
I ran `buildifier -v -mode=diff ./tensorflow/core/kernels/BUILD` but none of the lines I changed were indicated by the diff.

Reply:
Pardon, it does add one empty line to the file between the two virtual targets I added.  How would you like me to proceed?  Would you like me to run `buildifier` against all BUILD files I changed?

Reply:
I was wondering would `buildifier` help re-indent these `load`s. seems it doesn't. could you re-indent them?